frontend/expense-sharing/src/ExpenseSharingPage.js

```diff
@@ -0,0 +1,160 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+const ExpenseForm = ({ addExpense, userId, groupId }) => {
+  const [amount, setAmount] = useState('');
+  const [description, setDescription] = useState('');
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    const newExpense = {
+      amount,
+      description,
+      paid_by: userId, 
+      group_id: groupId, 
+      is_paid: false,
+    };
+    try {
+      const response = await axios.post('/add_expense', newExpense);
```

Comment:
Let's use `fetch` instead of `axios` to avoid overhead, it's native and more lightweight.